### DISCUSSION TITLE
Mixin dashboard not working on helm installation

### DISCUSSION THREAD

**Discussion by tejas-deshmukh-rupeek** [grafana/mimir#1896] [Help and support] [answered]
I have installed mimir helm chart and now trying to import dashboards but facing errors in no data on most of the panes.
No data for some of the metrics for example: cortex_distributor_received_samples to get samples per second.
Can someone help me

**Comment by pstibrany** [comment, marked as answer]
Hi. Mimir Dashboards use some metrics generated by rules. You can find the rules in [operations/mimir-mixin-compiled/rules.yaml](https://github.com/grafana/mimir/blob/main/operations/mimir-mixin-compiled/rules.yaml) file (compiled from [mimir mixin](https://github.com/grafana/mimir/tree/main/operations/mimir-mixin)).

You will need to install these rules into Prometheus that is scraping the Mimir metrics.

**Reply by pracucci** [reply]
Let me also link some related doc:
- https://grafana.com/docs/mimir/latest/operators-guide/visualizing-metrics/installing-dashboards-and-alerts/
- https://grafana.com/docs/mimir/latest/operators-guide/visualizing-metrics/requirements/

**Reply by tejas-deshmukh-rupeek** [reply]
Thanks @pstibrany and @pracucci, it helped a lot. Totally missed the recording rules part.
